src/debug.h

```diff
@@ -39,8 +39,10 @@ int checkCellhdxmax(const struct cell *c, int *depth);
 void dumpCells(const char *prefix, int super, int active, int mpiactive,
                int pactive, struct space *s, int rank, int step);
 
-#if defined(WITH_MPI) && (defined(HAVE_METIS) || defined(HAVE_PARMETIS))
-#include "metis.h"
+#if defined(WITH_MPI) && (defined(HAVE_METIS) || defined(HAVE_PARMETIS)) \
+    && !defined(HAVE_SCOTCH)
+typedef int64_t idx_t;
```

Comment:
This typedef is not needed. It will be defined in the METIS include and should match the integer type, i.e. 32 or 64 bits.

Reply:
Agreed. I should not be meddling with metis/parmetis - only code. 
(I always load the 64 bits version of both metis and parmetis, so I did not notice any error / warning during builds). Line 44 will not be present in the prod MR.